docker-compose.yaml

```diff
@@ -0,0 +1,11 @@
+version: "3.4"
+services:
+  docker:
+    image: homework25:latest
+    container_name: docker-compose
+    build: /home/authz/git
```

Comment:
lets use relative paths here. In order for developers to be able to run this service locally